guide/payments/transactions.md

```diff
@@ -0,0 +1,88 @@
+   caption = '3D payments and transaction lifecycle illustration by [Alexa Aker](https://alexaaker.com){:target="_blank" rel="nofollow"}'
+   width = 1600
+   height = 800
+   layout = "full-width"
+%}
+
+# Transactions overview
+
+Bitcoin is a peer-to-peer push-payment system. This means that you can send, or *push*, bitcoin to any address at your will, at any time, without passing through a third party.
+
+
+This is radically different from the legacy financial system, where it is often possible for others to *pull* and withdraw money from your account (utility companies, financial institutions, merchants, etc.). When you make a payment, it will pass through systems that might delay, control or block the payment. 
+
+## Transaction lifecycle
+
+Let's lay out the entire payment process.
+
+![](https://i.imgur.com/IzJLhXb.png)
+
+#### 1. Get recipient address
+You need a valid address to send bitcoin. This can be shared by the recipient as a QR code, in plain text or as a payment link.
+
+#### 2. Creation
+The wallet application guides the user through collecting the required information (address and amount), and any optional information (what coins to send, fee options).
+
+#### 3. Signing
+The transaction needs to be signed by the [private key(s)]({{ '/guide/glossary/#private-key' | relative_url }}) of the input address(es) to be valid. The signing is often done in the same application and at the same time as the creation, but this does not have to be the case.
+
+#### 4. Broadcasting
+The transaction is broadcasted to a Bitcoin node, normally the one the wallet is connected to.
+
+#### 5. Validation
+The receiving node checks that the transaction is valid. In practice this means confirming that it was signed by the private key of the relevant address.
+
+#### 6. Propagation
+Once validated, the node passes the transaction on to other nodes in the network - returning a success message to the originator. The transaction is now in the *memory pool*, and remains there until confirmed.
+
+#### 7. Confirmations
+At some point the transaction is included in a block that is added to the blockchain. This counts as a first confirmation. The recipient can be confident that they will receive the bitcoin, and may even receive a pre-emptive notification from their wallet. How long the first confirmation takes is dependent on many factors, principally - how many other transactions are waiting to be confirmed and how big a fee the transaction will pay. A new block is confirmed roughly every ten minutes.
+
+Each additional block that is accepted by the network counts as an extra confirmation, making the transaction more secure by making it harder to undo. When a transaction has 6 confirmations, it is considered final.
+
+---
+
+## Transaction structure
+
+The most common Bitcoin transactions contain instructions to move funds from one address to another. Other types of transactions include creating multi-sig wallets or recording data on the blockchain.
```

Comment:
Do we expect a person to know what a multi-sig is at this point, or should we refer them somewhere?

Reply:
Ye good catch on this. Setting up multisig happens off-chain there's no transaction besides the funding which is just as any normal payment to a bitcoin script.